workspace/scm.py

```diff
@@ -195,6 +195,12 @@ def merge_branch(branch, commit=None, squash=False, strategy=None):
         cmd.append('--strategy=' + strategy)
     if commit:
         cmd.append(commit)
+
+    if strategy:
+        current = current_branch()
+        message = f"Merge branch {branch} into {current} (using strategy {strategy})"
```

Comment:
we spoke offline about this, but also need to adjust the message if a commit was provided.